dockerfiles/Dockerfile.osm-crds

```diff
@@ -2,7 +2,7 @@ FROM --platform=$BUILDPLATFORM busybox:1.33 AS builder
 ARG TARGETPLATFORM
 # Talking to the internet in an arm64 container doesn't seem to work from a
 # amd64 Mac, so download the kubectl binary in a stage running the native arch.
-RUN wget https://dl.k8s.io/release/v1.22.2/bin/$TARGETPLATFORM/kubectl -O /bin/kubectl && \
+RUN wget https://dl.k8s.io/release/v1.23.5/bin/linux/amd64/kubectl -O /bin/kubectl && \
```

Comment:
This still needs `$TARGETPLATFORM` so arm64 builds work.

Reply:
oops, bad copy/pasta

Reply:
fixed now